msgid ""
msgstr ""
"Project-Id-Version: duplicity\n"
"Report-Msgid-Bugs-To: Kenneth Loafman <kenneth@loafman.com>\n"
"POT-Creation-Date: 2023-12-27 10:21-0600\n"
"PO-Revision-Date: 2023-12-27 17:28\n"
"Last-Translator: \n"
"Language-Team: German, Austria\n"
"Language: de_AT\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: duplicity\n"
"X-Crowdin-Project-ID: 539508\n"
"X-Crowdin-Language: de-AT\n"
"X-Crowdin-File: /main/po/duplicity.pot\n"
"X-Crowdin-File-ID: 6\n"

#: ../bin/duplicity:76
msgid "INT intercepted...exiting."
msgstr "INT abgefangen … wird beendet."

#: ../bin/duplicity:84
#, python-format
msgid "GPG error detail: %s"
msgstr "GPG-Fehlerdetail: %s"

#: ../bin/duplicity:91
#, python-format
msgid "User error detail: %s"
msgstr "Benutzerfehlerdetail: %s"

#: ../bin/duplicity:98
#, python-format
msgid "Backend error detail: %s"
msgstr "Fehlerdetail des Hintergrundprogramms: %s"

#: ../duplicity/tempdir.py:157
#, python-format
msgid "Using temporary directory %s"
msgstr "Temporäres Verzeichnis %s wird benutzt"

#: ../duplicity/tempdir.py:201
#, python-format
msgid "Registering (mktemp) temporary file %s"
msgstr "(mktemp) temporäre Datei %s wird registriert"

#: ../duplicity/tempdir.py:227
#, python-format
msgid "Registering (mkstemp) temporary file %s"
msgstr "(mkstemp) temporäre Datei %s wird registriert"

#: ../duplicity/tempdir.py:259
#, python-format
msgid "Forgetting temporary file %s"
msgstr "Temporäre Datei %s wird vergessen"

#: ../duplicity/tempdir.py:262
#, python-format
msgid "Attempt to forget unknown tempfile %s - this is probably a bug."
msgstr "Unbekannte temporäre Datei %s wird versucht zu vergessen - das ist wahrscheinlich ein Fehler."

#: ../duplicity/tempdir.py:281
#, python-format
msgid "Removing still remembered temporary file %s"
msgstr "Bereits gemerkte temporäre Datei %s wird entfernt"

#: ../duplicity/tempdir.py:284
#, python-format
msgid "Cleanup of temporary file %s failed"
msgstr "Entfernen der temporären Datei %s ist fehlgeschlagen"

#: ../duplicity/tempdir.py:290
#, python-format
msgid "Cleanup of temporary directory %s failed - this is probably a bug."
msgstr "Aufräumen des temporären Verzeichnisses %s fehlgeschlagen - das ist wahrscheinlich ein Fehler."

#: ../duplicity/backend.py:92
#, python-format
msgid "Import of %s %s"
msgstr "Import von %s %s"

#: ../duplicity/backend.py:197
#, python-format
msgid "Could not initialize backend: %s"
msgstr "Hintergrundprogramm konnte nicht initialisiert werden: %s"

#: ../duplicity/backend.py:388
#, python-format
msgid "Backtrace of previous error: %s"
msgstr "Rückverfolgung des vorangegangenen Fehlers: %s"

#: ../duplicity/backend.py:407
#, python-format
msgid "Giving up after %s attempts. %s: %s"
msgstr "Nach %s Versuchen wird aufgegeben. %s: %s"

#: ../duplicity/backend.py:413
#, python-format
msgid "Attempt of %s Nr. %s failed. %s: %s"
msgstr ""

#: ../duplicity/backend.py:512
#, python-format
msgid "Reading results of '%s'"
msgstr "Ergebnisse von »%s« werden gelesen"

#: ../duplicity/backend.py:539
#, python-format
msgid "Writing %s"
msgstr "%s wird geschrieben"

#: ../duplicity/backend.py:580
#, python-format
msgid "File %s not found locally after get from backend"
msgstr "Datei %s konnte nicht lokal gefunden werden, nach Erhalt vom Hintergrundprogramm."

#: ../duplicity/manifest.py:95
#, python-format
msgid "Fatal Error: Backup source host has changed.\n"
"Current hostname: %s\n"
"Previous hostname: %s"
msgstr "Kritischer Fehler: Sicherungsquellrechner hat sich geändert.\n"
"Aktueller Rechnername: %s\n"
"Vorheriger Rechnername: %s"

#: ../duplicity/manifest.py:102
msgid "Fatal Error: Backup source directory has changed.\n"
msgstr ""

#: ../duplicity/manifest.py:115
msgid "Aborting because you may have accidentally tried to backup two different data sets to the same remote location, or using the same archive directory.  If this is not a mistake, use the --allow-source-mismatch switch to avoid seeing this message"
msgstr "Abbruch, weil Sie möglicherweise versehentlich versucht haben, zwei verschiedene Datensätze zum gleichen entfernten Ort oder zum selben Archivverzeichnis zu sichern. Wenn dies kein Fehler ist, verwenden Sie den »allow-source-mismatch«-Parameter, um diese Meldung auszuschalten."

#: ../duplicity/manifest.py:221
#, python-format
msgid "Found manifest volume %s"
msgstr "Manifest-Volumen %s gefunden"

#: ../duplicity/manifest.py:228
#, python-format
msgid "Found %s volumes in manifest"
msgstr "%s Volumen in Manifest gefunden"

#: ../duplicity/manifest.py:249
msgid "Manifest file '{self.fh.base if self.fh else ''}' is corrupt: "
msgstr ""

#: ../duplicity/manifest.py:268
msgid "Manifests not equal because different volume numbers"
msgstr "Manifests sind ungleich aufgrund verschiedener Datenträgerzahlen"

#: ../duplicity/manifest.py:273
msgid "Manifests not equal because volume lists differ"
msgstr "Manifests sind ungleich aufgrund verschiedenen Datenträgerlisten"

#: ../duplicity/manifest.py:277
msgid "Manifests not equal because hosts or directories differ"
msgstr "Manifests sind ungleich aufgrund Rechner- oder Verzeichnisunterschieden"

#: ../duplicity/manifest.py:438
msgid "Warning, found extra Volume identifier"
msgstr "Achtung, zusätzliche Datenträgerbezeichnung gefunden"

#: ../duplicity/manifest.py:464
msgid "Other is not VolumeInfo"
msgstr "Anderes ist keine Datenträgerinfo"

#: ../duplicity/manifest.py:467
msgid "Volume numbers don't match"
msgstr "Datenträgerzahlen stimmen nicht überein"

#: ../duplicity/manifest.py:470
msgid "start_indicies don't match"
msgstr "Startindizes stimmen nicht überein"

#: ../duplicity/manifest.py:473
msgid "end_index don't match"
msgstr "Endindex stimmt nicht überein"

#: ../duplicity/manifest.py:478
msgid "Hashes don't match"
msgstr "Prüfsummen stimmen nicht überein"

#: ../duplicity/util.py:105 ../duplicity/dup_main.py:906
#, python-format
msgid "IGNORED_ERROR: Warning: ignoring error as requested: %s: %s"
msgstr "VERNACHLÄSSIGTER FEHLER: Achtung: Fehler wird wie verlangt vernachlässigt: %s: %s"

#: ../duplicity/util.py:170
#, python-format
msgid "Releasing lockfile %s"
msgstr "Sperrdatei »%s« wird freigegeben"

#: ../duplicity/backends/pydrivebackend.py:203
#, python-format
msgid "PyDrive backend: multiple files called '%s'."
msgstr "PyDrive-Hintergrundprogramm: mehrere Dateien aufgerufen »%s«."

#: ../duplicity/backends/multibackend.py:100
#, python-format
msgid "MultiBackend: Could not parse query string %s: %s "
msgstr "MultiBackend: Konnte den angefragten String %s nicht auswerten: %s "

#: ../duplicity/backends/multibackend.py:110
#, python-format
msgid "MultiBackend: Invalid query string %s: more than one value for %s"
msgstr "MultiBackend: angefragter String %s ist nicht gültig: mehr als einen Wert für %s"

#: ../duplicity/backends/multibackend.py:116
#, python-format
msgid "MultiBackend: Invalid query string %s: unknown parameter %s"
msgstr "MultiBackend: angefragter String %s ist nicht gültig: unbekannter Parameter %s"

#: ../duplicity/backends/multibackend.py:167
#: ../duplicity/backends/multibackend.py:174
#, python-format
msgid "MultiBackend: illegal value for %s: %s"
msgstr "MultiBackend: nicht erlaubter Wert für %s: %s"

#: ../duplicity/backends/multibackend.py:186
#, python-format
msgid "MultiBackend: Url %s"
msgstr "MehrfachHintergrundprogramm: Adresse %s"

#: ../duplicity/backends/multibackend.py:189
#, python-format
msgid "MultiBackend: Could not load config file %s: %s "
msgstr "MehrfachHintergrundprogramm: Konfigurationsdatei konnte nicht geladen werden %s: %s "

#: ../duplicity/backends/multibackend.py:196
#, python-format
msgid "MultiBackend: use store %s"
msgstr "MehrfachHintergrundprogramm: Speicher %s benutzen"

#: ../duplicity/backends/multibackend.py:200
#, python-format
msgid "MultiBackend: set env %s = %s"
msgstr "MehrfachHintergrundprogramm: env %s = %s einstellen"

#: ../duplicity/backends/multibackend.py:214
#, python-format
msgid "Multibackend: register affinity for prefix %s"
msgstr "Multibackend: Registeraffinität für Präfix %s"

#: ../duplicity/backends/multibackend.py:257
#, python-format
msgid "MultiBackend: _put: write to store #%s (%s)"
msgstr "MehrfachHintergrundprogramm: _put: Zum Speicher schreiben #%s (%s)"

#: ../duplicity/backends/multibackend.py:272
#, python-format
msgid "MultiBackend: failed to write to store #%s (%s), try #%s, Exception: %s"
msgstr "MehrfachHintergrundprogramm: Es konnte nicht zum Speicher geschrieben werden #%s (%s), bitte #%s versuchen, Ausnahmefall: %s"

#: ../duplicity/backends/multibackend.py:286
#, python-format
msgid "MultiBackend: failed to write %s. Aborting process."
msgstr "MehrfachHintergrundprogramm: Schreiben von %s ist fehlgeschlagen. Vorgang wird abgebrochen."

#: ../duplicity/backends/multibackend.py:294
#, python-format
msgid "MultiBackend: failed to write %s. Tried all backing stores and none succeeded"
msgstr "MehrfachHintergrundprogramm: %s konnte nicht geschrieben werden. Es wurden alle Hintergrundspeicher versucht, aber ohne Erfolg"

#: ../duplicity/backends/multibackend.py:315
#, python-format
msgid "MultiBackend: failed to get %s to %s from %s"
msgstr "MehrfachHintergrundprogramm: %s konnte nicht zu %s von %s gelangen"

#: ../duplicity/backends/multibackend.py:320
#, python-format
msgid "MultiBackend: failed to get %s. Tried all backing stores and none succeeded"
msgstr "MehrfachHintergrundprogramm: %s konnte nicht erhalten werden. Es wurden alle Hintergrundspeicher versucht, aber ohne Erfolg"

#: ../duplicity/backends/multibackend.py:330
#, python-format
msgid "MultiBackend: %s: %d files"
msgstr "MultiBackend: %s: %d Dateien"

#: ../duplicity/backends/multibackend.py:334
msgid "Exception during list of {s.backend.parsed_url.strip_auth()}: "
msgstr ""

#: ../duplicity/backends/multibackend.py:342
#, python-format
msgid "MultiBackend: combined list: %s"
msgstr "MehrfachHintergrundprogramm: kombinierte Liste: %s"

#: ../duplicity/backends/multibackend.py:371
#: ../duplicity/backends/multibackend.py:398
#, python-format
msgid "MultiBackend: failed to delete %s. Tried all backing stores and none succeeded"
msgstr "MehrfachHintergrundprogramm: %s konnte nicht gelöscht werden. Es wurden alle Hintergrundspeicher versucht, aber ohne Erfolg"

#: ../duplicity/backends/giobackend.py:108
#, python-format
msgid "Connection failed, please check your password: %s"
msgstr "Verbindung fehlgeschlagen, bitte Passwort prüfen: %s"

#: ../duplicity/backends/onedrivebackend.py:90
#, python-format
msgid "Attempt of initialize_oauth2_session Nr. %s failed. %s: %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:67
msgid "Missing socket or ssl python modules."
msgstr "Es fehlt ein Socket oder ein SSL Python Modul."

#: ../duplicity/backends/webdavbackend.py:87
#, python-format
msgid "Cacert database file '%s' is not readable."
msgstr "Die Cacert Datenbankdatei '%s' ist nicht lesbar."

#: ../duplicity/backends/webdavbackend.py:149
#, python-format
msgid "Using WebDAV host %s port %s"
msgstr "WebDAV-Rechner %s Anschluss (Port) %s wird verwendet"

#: ../duplicity/backends/webdavbackend.py:150
#, python-format
msgid "Using WebDAV directory %s"
msgstr "WebDAV-Verzeichnis %s wird verwendet"

#: ../duplicity/backends/webdavbackend.py:180
#, python-format
msgid "WebDAV create connection on '%s'"
msgstr "WebDAV erzeugt eine Verbindung unter '%s'"

#: ../duplicity/backends/webdavbackend.py:191
#, python-format
msgid "WebDAV Unknown URI scheme: %s"
msgstr "WebDAV unbekanntes URI Schema: %s"

#: ../duplicity/backends/webdavbackend.py:210
#, python-format
msgid "WebDAV %s %s request with headers: %s "
msgstr "WebDAV %s %s Anfrage mit Headern: %s "

#: ../duplicity/backends/webdavbackend.py:211
#: ../duplicity/backends/webdavbackend.py:234
#, python-format
msgid "WebDAV data length: %s "
msgstr "WebDAV Datenlänge: %s "

#: ../duplicity/backends/webdavbackend.py:214
#, python-format
msgid "WebDAV response status %s with reason '%s'."
msgstr "WebDAV Antwortstatus %s mit dem Grund '%s'."

#: ../duplicity/backends/webdavbackend.py:220
#, python-format
msgid "WebDAV redirect to: %s "
msgstr "WebDAV weitersenden an: %s "

#: ../duplicity/backends/webdavbackend.py:222
msgid "WebDAV redirected 10 times. Giving up."
msgstr "WebDAV leitete 10 mal um. Gebe auf."

#: ../duplicity/backends/webdavbackend.py:227
msgid "WebDAV missing location header in redirect response."
msgstr "WebDAV fehlender Lokalisierungs-Header in redirect response."

#: ../duplicity/backends/webdavbackend.py:232
msgid "WebDAV retry request with authentification headers."
msgstr "WebDAV versucht erneut, eine Anforderung mit Authentifizierungs-Headern zu stellen."

#: ../duplicity/backends/webdavbackend.py:233
#, python-format
msgid "WebDAV %s %s request2 with headers: %s "
msgstr "WebDAV %s %s Anfrage2 mit Headern: %s "

#: ../duplicity/backends/webdavbackend.py:237
#, python-format
msgid "WebDAV response2 status %s with reason '%s'."
msgstr "WebDAV Antwort2 Status %s mit Grund »%s«."

#: ../duplicity/backends/webdavbackend.py:256
msgid "python-kerberos needed to use kerberos                           authorization, falling back to basic auth."
msgstr "python-kerberos benötigt Kerberos-Legitimierung, auf die Basislegitimierung wird zurückgegriffen."

#: ../duplicity/backends/webdavbackend.py:264
#, python-format
msgid "Kerberos authorization failed: %s.                          Falling back to basic auth."
msgstr "Kerberos-Legitimierung ist fehlgeschlagen: %s. Auf die Basislegitimierung wird zurückgegriffen."

#: ../duplicity/backends/webdavbackend.py:365
#, python-format
msgid "Creating missing directory %s"
msgstr "Fehlendes Verzeichnis »%s« wird erstellt"

#: ../duplicity/backends/webdavbackend.py:369
#, python-format
msgid "WebDAV MKCOL %s failed: %s %s"
msgstr "WebDAV MKCOL %s fehlgeschlagen: %s %s"

#: ../duplicity/backends/webdavbackend.py:381
#, python-format
msgid "WebDAV path decoding and translation: %s -> %s"
msgstr "WebDAV Pfad-Dekodierung und Übersetzung: %s -> %s"

#: ../duplicity/backends/webdavbackend.py:421
#, python-format
msgid "WebDAV GET Bad status code %s reason %s."
msgstr "WebDAV GET Falscher Statuscode %s Grund %s."

#: ../duplicity/backends/webdavbackend.py:442
#, python-format
msgid "WebDAV PUT Bad status code %s reason %s."
msgstr "WebDAV PUT Falscher Statuscode %s Grund %s."

#: ../duplicity/backends/webdavbackend.py:461
#, python-format
msgid "WebDAV DEL Bad status code %s reason %s."
msgstr "WebDAV DEL Falscher Statuscode %s Grund %s."

#. Used in usage help to represent a Unix-style path name. Example:
#. --archive-dir <path>
#: ../duplicity/cli_data.py:102 ../duplicity/cli_data.py:162
#: ../duplicity/cli_data.py:196 ../duplicity/cli_data.py:243
#: ../duplicity/cli_data.py:346 ../duplicity/cli_data.py:364
#: ../duplicity/cli_data.py:524 ../duplicity/cli_data.py:706
#: ../duplicity/cli_data.py:932
msgid "path"
msgstr "Pfad"

#: ../duplicity/cli_data.py:115
msgid "Hot|Cool|Archive"
msgstr "Hot|Cool|Archiv"

#. Used in usage help to represent a desired number of
#. something. Example:
#. --num-retries <number>
#: ../duplicity/cli_data.py:120 ../duplicity/cli_data.py:126
#: ../duplicity/cli_data.py:134 ../duplicity/cli_data.py:168
#: ../duplicity/cli_data.py:432 ../duplicity/cli_data.py:448
#: ../duplicity/cli_data.py:495 ../duplicity/cli_data.py:512
#: ../duplicity/cli_data.py:518 ../duplicity/cli_data.py:536
#: ../duplicity/cli_data.py:602 ../duplicity/cli_data.py:610
#: ../duplicity/cli_data.py:662 ../duplicity/cli_data.py:743
#: ../duplicity/cli_data.py:919
msgid "number"
msgstr "Zahl"

#. Used in usage help. Example:
#. --timeout <seconds>
#: ../duplicity/cli_data.py:146 ../duplicity/cli_data.py:713
#: ../duplicity/cli_data.py:945
msgid "seconds"
msgstr "Sekunden"

#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#: ../duplicity/cli_data.py:184 ../duplicity/cli_data.py:190
#: ../duplicity/cli_data.py:358 ../duplicity/cli_data.py:669
#: ../duplicity/cli_data.py:911
msgid "gpg-key-id"
msgstr "GPG-Schlüsselkennung"

#. Used in usage help to represent a "glob" style pattern for
#. matching one or more files, as described in the documentation.
#. Example:
#. --exclude <shell_pattern>
#: ../duplicity/cli_data.py:201 ../duplicity/cli_data.py:386
#: ../duplicity/cli_data.py:950
msgid "shell_pattern"
msgstr ""

#. Used in usage help to represent the name of a file. Example:
#. --log-file <filename>
#: ../duplicity/cli_data.py:213 ../duplicity/cli_data.py:219
#: ../duplicity/cli_data.py:279 ../duplicity/cli_data.py:392
#: ../duplicity/cli_data.py:908
msgid "filename"
msgstr "Dateiname"

#. Used in usage help to represent a time spec for a previous
#. point in time, as described in the documentation. Example:
#. duplicity remove-older-than time [options] target_url
#: ../duplicity/cli_data.py:225 ../duplicity/cli_data.py:549
#: ../duplicity/cli_data.py:976
msgid "time"
msgstr "Zeit"

#: ../duplicity/cli_data.py:237 ../duplicity/cli_data.py:398
msgid "regex"
msgstr ""

#: ../duplicity/cli_data.py:340
msgid "interval"
msgstr ""

#. Used in usage help. (Should be consistent with the "Options:"
#. header.) Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:352 ../duplicity/cli_data.py:506
#: ../duplicity/cli_data.py:555 ../duplicity/cli_data.py:685
#: ../duplicity/cli_data.py:923 ../duplicity/argparse311.py:1671
msgid "options"
msgstr "Optionen"

#: ../duplicity/cli_data.py:381
msgid "imap_mailbox"
msgstr "IMAP-Postfach"

#: ../duplicity/cli_data.py:406
msgid "If set, an extra file with statistics in json format will be stored with each backup. It includes the statistics printed out on stdout by default. With 'collection-status' it returs these stats."
msgstr ""

#: ../duplicity/cli_data.py:414
msgid "file_descriptor"
msgstr "Dateideskriptor"

#: ../duplicity/cli_data.py:421
msgid "log_filename"
msgstr ""

#: ../duplicity/cli_data.py:454
msgid "backup name"
msgstr "Sicherungsname"

#: ../duplicity/cli_data.py:561
msgid "s3_endpoint_url"
msgstr ""

#: ../duplicity/cli_data.py:629
msgid "s3_kms_key_id"
msgstr ""

#: ../duplicity/cli_data.py:635
msgid "s3_kms_grant"
msgstr ""

#: ../duplicity/cli_data.py:641
msgid "s3_region_name"
msgstr ""

#: ../duplicity/cli_data.py:647
msgid "policy"
msgstr "Richtlinie"

#. noun
#: ../duplicity/cli_data.py:652 ../duplicity/cli_data.py:657
#: ../duplicity/cli_data.py:897
msgid "command"
msgstr "Befehl"

#: ../duplicity/cli_data.py:692
msgid "pem formatted bundle of certificate authorities"
msgstr "PEM-formatiertes Paket von Zertifizierungsstellen"

#: ../duplicity/cli_data.py:697
msgid "path to a folder with certificate authority files"
msgstr "Pfad zu einem Verzeichnis mit Dateien der Zertifizierungsstellen"

#. abbreviation for "character" (noun)
#: ../duplicity/cli_data.py:719 ../duplicity/cli_data.py:895
msgid "char"
msgstr "Zeichen"

#: ../duplicity/cli_data.py:732
msgid "verb"
msgstr ""

#: ../duplicity/cli_data.py:750
msgid "extra headers for Webdav, like 'Cookie,name=value'"
msgstr ""

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other_host[:port]//absolute_path
#: ../duplicity/cli_data.py:886
msgid "absolute_path"
msgstr "absoluter_Pfad"

#. Used in usage help. Example:
#. tahoe://alias/some_dir
#: ../duplicity/cli_data.py:889
msgid "alias"
msgstr "Alias"

#. Used in help to represent a "bucket name" for Amazon Web
#. Services' Simple Storage Service (S3). Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/cli_data.py:893
msgid "bucket_name"
msgstr "Bucket_Name"

#. Used in usage help to represent the name of a container in
#. Amazon Web Services' Cloudfront. Example:
#. cf+http://container_name
#: ../duplicity/cli_data.py:901
msgid "container_name"
msgstr "Containername"

#. noun
#: ../duplicity/cli_data.py:903
msgid "count"
msgstr "Anzahl"

#. Used in usage help to represent the name of a file directory
#: ../duplicity/cli_data.py:905
msgid "directory"
msgstr "Verzeichnis"

#. Used in usage help, e.g. to represent the name of a code
#. module. Example:
#. rsync://user[:password]@other.host[:port]::/module/some_dir
#: ../duplicity/cli_data.py:915
msgid "module"
msgstr "Modul"

#. Used in usage help to represent an internet hostname. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:926
msgid "other.host"
msgstr "anderer.Rechner"

#. Used in usage help. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:929
msgid "password"
msgstr "Passwort"

#. Used in usage help to represent a TCP port number. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:935
msgid "port"
msgstr "Port"

#. Used in usage help. This represents a string to be used as a
#. prefix to names for backup files created by Duplicity. Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/cli_data.py:939
msgid "prefix"
msgstr "Präfix"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other.host[:port]/relative_path
#: ../duplicity/cli_data.py:942
msgid "relative_path"
msgstr "relativer_Pfad"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. Example:
#. file:///some_dir
#: ../duplicity/cli_data.py:954
msgid "some_dir"
msgstr "irgendein_Verzeichnis"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory where files will be
#. coming FROM. Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:959
msgid "source_path"
msgstr ""

#. Used in usage help to represent a URL files will be coming
#. FROM. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/cli_data.py:963
msgid "source_url"
msgstr "Quelladresse"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. where files will be
#. going TO. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/cli_data.py:968
msgid "target_dir"
msgstr "Zielverzeichnis"

#. Used in usage help to represent a URL files will be going TO.
#. Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:972
msgid "target_url"
msgstr "Zieladresse"

#. Used in usage help to represent a user name (i.e. login).
#. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:980
msgid "user"
msgstr "Benutzer"

#. account id for b2. Example: b2://account_id@bucket/
#: ../duplicity/cli_data.py:982
msgid "account_id"
msgstr "Kontokennung"

#. application_key for b2.
#. Example: b2://account_id:application_key@bucket/
#: ../duplicity/cli_data.py:985
msgid "application_key"
msgstr "Anwendungsschlüssel"

#. remote name for rclone.
#. Example: rclone://remote:/some_dir
#: ../duplicity/cli_data.py:988 ../duplicity/dup_collections.py:565
msgid "remote"
msgstr "Entfernt"

#: ../duplicity/cli_data.py:992
msgid "Backends and their URL formats:"
msgstr "Hintergrundprogramme und deren Adressformate:"

#: ../duplicity/diffdir.py:102 ../duplicity/diffdir.py:401
#, python-format
msgid "Error %s getting delta for %s"
msgstr "Fehler %s beim Erhalt des Delta für %s"

#: ../duplicity/diffdir.py:116
#, python-format
msgid "Getting delta of %s and %s"
msgstr "Delta von %s und %s wird erhalten"

#: ../duplicity/diffdir.py:160
#, python-format
msgid "A %s"
msgstr "Ein %s"

#: ../duplicity/diffdir.py:168
#, python-format
msgid "M %s"
msgstr ""

#: ../duplicity/diffdir.py:191
#, python-format
msgid "Comparing %s and %s"
msgstr "%s und %s werden verglichen"

#: ../duplicity/diffdir.py:204
#, python-format
msgid "D %s"
msgstr ""

#: ../duplicity/librsync.py:194
msgid "basis_file must be a (true) file or an object whose file attribute is the underlying true file object"
msgstr "basis_file muss eine (echte) Datei oder ein Objekt sein, dessen Datei-Attribut das zugrundeliegende Dateiobjekt ist"

#: ../duplicity/dup_time.py:54
#, python-format
msgid "Bad interval string \"%s\"\n\n"
"Intervals are specified like 2Y (2 years) or 2h30m (2.5 hours).  The\n"
"allowed special characters are s, m, h, D, W, M, and Y.  See the man\n"
"page for more information."
msgstr "Ungültige Intervallzeichenkette »%s«\n\n"
"Intervalle sind angegeben wie 2Y (2 Jahre) oder 2h30m (2,5 Stunden).\n"
"Die erlaubten Spezialzeichen sind s, m, h, D, W, M, und Y.\n"
"Weitere Informationen sind im Handbuch zu finden."

#: ../duplicity/dup_time.py:63
#, python-format
msgid "Bad time string \"%s\"\n\n"
"The acceptible time strings are intervals (like \"3D64s\"), w3-datetime\n"
"strings, like \"2002-04-26T04:22:01-07:00\" (strings like\n"
"\"2002-04-26T04:22:01\" are also acceptable - duplicity will use the\n"
"current time zone), or ordinary dates like 2/4/1997 or 2001-04-23\n"
"(various combinations are acceptable, but the month always precedes\n"
"the day)."
msgstr "Ungültige Zeitzeichenkette »%s«\n\n"
"Gültige Zeitzeichenkette sind Intervalle (wie »3D46s«), w3-Datum-Zeitzeichenketten,\n"
"wie »2002-04-26T04:22:01-07:00« (Zeichenketten wie »2002-04-26T04:22:01« sind\n"
"auch gültig - duplicity wird die aktuelle Zeitzone nutzen, oder gewöhnliche Daten\n"
"wie 2/4/1997 oder 2001-04-23 (verschiedene Kombinationen sind gültig, aber vor\n"
"dem Monat kommt immer der Tag)."

#: ../duplicity/cli_main.py:155
msgid "No valid action found. Will imply 'backup' because a path source was given and target is a url location."
msgstr ""

#: ../duplicity/cli_main.py:163
msgid "No valid action found. Will imply 'restore' because url source was given and target is a local path."
msgstr ""

#: ../duplicity/cli_main.py:170
#, python-brace-format
msgid "Invalid '{remainder[0]}' action and cannot be implied from the "
msgstr ""

#: ../duplicity/cli_main.py:291
#, python-brace-format
msgid "GPG binary is {config.gpg_binary}, version {gpg_version}"
msgstr ""

#: ../duplicity/cli_main.py:327
#, python-brace-format
msgid "Using archive dir: {config.archive_dir_path.uc_name}"
msgstr ""

#: ../duplicity/cli_main.py:328
#, python-brace-format
msgid "Using backup name: {config.backup_name}"
msgstr ""

#: ../duplicity/dup_main.py:113
msgid "Reuse configured PASSPHRASE as SIGN_PASSPHRASE"
msgstr "Eingestellte PASSPHRASE als SIGN_PASSPHRASE benutzen"

#: ../duplicity/dup_main.py:124
msgid "Reuse configured SIGN_PASSPHRASE as PASSPHRASE"
msgstr "Eingestellte SIGN_PASSPHRASE als PASSPHRASE benutzen"

#: ../duplicity/dup_main.py:168
msgid "PASSPHRASE variable not set, asking user."
msgstr "Variable PASSPHRASE nicht eingestellt, Benutzer wird gefragt."

#: ../duplicity/dup_main.py:193
msgid "Retype passphrase for signing key to confirm: "
msgstr "Zur Bestätigung bitte die Passphrase für die Schlüsselsignierung wiederholen: "

#: ../duplicity/dup_main.py:195
msgid "Retype passphrase for decryption to confirm: "
msgstr ""

#: ../duplicity/dup_main.py:199
msgid "First and second passphrases do not match!  Please try again."
msgstr "Erste und zweite Passphrase stimmen nicht überein! Bitte nochmal versuchen."

#: ../duplicity/dup_main.py:210
msgid "Cannot use empty passphrase with symmetric encryption!  Please try again."
msgstr "Bei symmetrischer Verschlüsselung darf keine leere Passphrase benutzt werden! Bitte nochmal versuchen."

#: ../duplicity/dup_main.py:270
#, python-format
msgid "File %s complete in backup set.\n"
"Continuing restart on file %s."
msgstr "Datei %s ist komplett im Sicherungssatz.\n"
"Nach Neustart erfolgt Fortsetzung mit Datei %s."

#: ../duplicity/dup_main.py:280
#, python-format
msgid "File %s missing in backup set.\n"
"Continuing restart on file %s."
msgstr "Datei %s fehlt im Sicherungssatz.\n"
"Beim Neustart wird fortgefahren mit Datei %s."

#: ../duplicity/dup_main.py:335
#, python-format
msgid "%s Remote filesize %d for %s does not match local size %d, retrying."
msgstr ""

#: ../duplicity/dup_main.py:342
#, python-format
msgid "File %s was corrupted during upload."
msgstr "Datei %s wurde während des Hochladens beschädigt."

#: ../duplicity/dup_main.py:382
msgid "Restarting backup, but current encryption settings do not match original settings"
msgstr "Datensicherung wird neu gestartet, die aktuellen Verschlüsselungseinstellungen entsprechen jedoch nicht den ursprünglichen Einstellungen"

#: ../duplicity/dup_main.py:404
msgid "Skipping encryption validation due to glacier/deep storage"
msgstr ""

#: ../duplicity/dup_main.py:408
#, python-format
msgid "Restarting after volume %s, file %s, block %s"
msgstr "Neustart nach Datenträger %s, Datei %s, Block %s"

#: ../duplicity/dup_main.py:471
msgid "Skipped volume upload, as effectivly nothing has changed"
msgstr ""

#: ../duplicity/dup_main.py:485
#, python-format
msgid "Processed volume %d"
msgstr "Datenträger %d wurde verarbeitet"

#: ../duplicity/dup_main.py:653
msgid "Fatal Error: Unable to start incremental backup.  Old signatures not found and incremental specified"
msgstr "Schwerwiegender Fehler: schrittweise Sicherung konnte nicht gestartet werden. Alte Signaturen nicht gefunden und schrittweise Sicherung ausgewählt."

#: ../duplicity/dup_main.py:659
msgid "No signatures found, switching to full backup."
msgstr "Keine Signaturen gefunden, vollständige Sicherung wird durchgeführt."

#: ../duplicity/dup_main.py:673
msgid "Backup Statistics"
msgstr "Sicherungsstatistiken"

#: ../duplicity/dup_main.py:813
#, python-format
msgid "%s not found in archive - no files restored."
msgstr "%s wurde nicht im Archiv gefunden - keine Dateien wiederhergestellt."

#: ../duplicity/dup_main.py:817
msgid "No files found in archive - nothing restored."
msgstr "Keine Dateien im Archiv - nichts wiederhergestellt."

#: ../duplicity/dup_main.py:853
#, python-format
msgid "Processed volume %d of %d"
msgstr "Datenträger %d von %d wurde verarbeitet"

#: ../duplicity/dup_main.py:896
#, python-format
msgid "Invalid data - %s hash mismatch for file:"
msgstr "Ungültige Daten – %s-Hash stimmt nicht für die Datei:"

#: ../duplicity/dup_main.py:898
#, python-format
msgid "Calculated hash: %s"
msgstr "Erstellter Hash: %s"

#: ../duplicity/dup_main.py:899
#, python-format
msgid "Manifest hash: %s"
msgstr "Manifest-Hashcode: %s"

#: ../duplicity/dup_main.py:952
#, python-format
msgid "Volume was signed by key %s, not %s"
msgstr "Datenträger wurde von Schlüssel %s, anstatt von %s unterschrieben"

#: ../duplicity/dup_main.py:984
#, python-format
msgid "Verify complete: %s, %s."
msgstr "Überprüfung vollständig: %s, %s."

#: ../duplicity/dup_main.py:985
#, python-format
msgid "%d file(s) compared"
msgstr ""

#: ../duplicity/dup_main.py:985
#, python-format
msgid "%d difference(s) found"
msgstr ""

#: ../duplicity/dup_main.py:1004
msgid "No extraneous files found, nothing deleted in cleanup."
msgstr "Keine überflüssigen Dateien gefunden, es wurde nichts gelöscht."

#: ../duplicity/dup_main.py:1019
msgid "Found the following file(s) to delete:"
msgstr ""

#: ../duplicity/dup_main.py:1023
msgid "Run duplicity again with the --force option to actually delete."
msgstr "Bitte duplicity noch einmal mit der »--force«-Option ausführen, um tatsächlich zu löschen."

#: ../duplicity/dup_main.py:1069
msgid "There are backup set(s) at time(s):"
msgstr "Es bestehen Sicherungssätze zu folgenden Zeitpunkten:"

#: ../duplicity/dup_main.py:1071
msgid "Which can't be deleted because newer sets depend on them."
msgstr "Welche nicht gelöscht werden können, weil neuere Sicherungssätze davon abhängen."

#: ../duplicity/dup_main.py:1078
msgid "Current active backup chain is older than specified time.  However, it will not be deleted.  To remove all your backups, manually purge the repository."
msgstr "Aktuell aktive Sicherungskette ist älter als der angegebene Zeitpunkt. Sie wird nicht gelöscht. Um alle Sicherungen zu entfernen, bitte die Paketquelle manuell löschen."

#: ../duplicity/dup_main.py:1096
msgid "No old backup sets found, nothing deleted."
msgstr "Keine alten Sicherungssätze gefunden, nichts gelöscht."

#: ../duplicity/dup_main.py:1099
msgid "Deleting backup chain(s) at time:"
msgstr ""

#: ../duplicity/dup_main.py:1108
#, python-format
msgid "Deleting any incremental signature chain rooted at %s"
msgstr "Jede schrittweise Signaturkette die bei %s verankert ist wird gelöscht"

#: ../duplicity/dup_main.py:1110
#, python-format
msgid "Deleting any incremental backup chain rooted at %s"
msgstr "Jede schrittweise Sicherungskette die bei %s verankert ist wird gelöscht"

#: ../duplicity/dup_main.py:1113
#, python-format
msgid "Deleting complete signature chain %s"
msgstr "Vollständige Signaturkette wird gelöscht: %s"

#: ../duplicity/dup_main.py:1115
#, python-format
msgid "Deleting complete backup chain %s"
msgstr "Vollständige Sicherungskette wird gelöscht: %s"

#: ../duplicity/dup_main.py:1122
msgid "Found old backup chain(s) at the following time:"
msgstr ""

#: ../duplicity/dup_main.py:1126
msgid "Rerun command with --force option to actually delete."
msgstr "Bitte Befehl erneut mit »--force«-Option ausführen, um tatsächlich zu löschen."

#: ../duplicity/dup_main.py:1229
#, python-format
msgid "Deleting local %s (not authoritative at backend)."
msgstr "Lokal %s wird gelöscht (keine Auswirkungen am Hintergrundprogramm)."

#: ../duplicity/dup_main.py:1233
#, python-format
msgid "Unable to delete %s: %s"
msgstr "%s kann nicht gelöscht werden: %s"

#: ../duplicity/dup_main.py:1267 ../duplicity/dup_temp.py:296
#, python-format
msgid "Failed to read %s: %s"
msgstr "%s konnte nicht gelesen werden: %s"

#: ../duplicity/dup_main.py:1279
#, python-format
msgid "Copying %s to local cache."
msgstr "%s wird zum lokalen Puffer kopiert."

#: ../duplicity/dup_main.py:1327
msgid "Local and Remote metadata are synchronized, no sync needed."
msgstr "Lokale und entfernte Metadaten sind bereits synchron, kein Abgleich benötigt."

#: ../duplicity/dup_main.py:1332
msgid "Synchronizing remote metadata to local cache..."
msgstr "Entfernte Metadaten werden zum lokalen Puffer synchronisiert …"

#: ../duplicity/dup_main.py:1346
msgid "Sync would copy the following from remote to local:"
msgstr "Synchronisation würde die folgenden Daten vom entfernten Rechner hierher kopieren:"

#: ../duplicity/dup_main.py:1352
msgid "Sync would remove the following spurious local files:"
msgstr "Synchronisation würde die folgenden überflüssigen lokalen Dateien entfernen:"

#: ../duplicity/dup_main.py:1398
msgid "Unable to get free space on temp."
msgstr "Der temporäre Speicherplatz ist belegt."

#: ../duplicity/dup_main.py:1405
#, python-format
msgid "Temp space has %d available, backup needs approx %d."
msgstr "%d temporärer Speicherplatz ist verfügbar, die Sicherung benötigt ungefähr %d."

#: ../duplicity/dup_main.py:1409
#, python-format
msgid "Temp has %d available, backup will use approx %d."
msgstr "%d temporärer Speicherplatz ist verfügbar, die Sicherung wird ungefähr %d nutzen."

#: ../duplicity/dup_main.py:1416
msgid "Unable to get max open files."
msgstr "Der Höchstwert für offene Dateien konnte nicht festgestellt werden."

#: ../duplicity/dup_main.py:1421
#, python-format
msgid "Max open files of %s is too low, should be >= 1024.\n"
"Use 'ulimit -n 1024' or higher to correct.\n"
msgstr "Limit für gleichzeitig offene Dateien von %s ist zu niedrig, sollte >=1024 sein.\n"
"Bitte »ulimit -n 1024« ausführen, um dieses zu berichtigen.\n"

#: ../duplicity/dup_main.py:1477
msgid "RESTART: The first volume failed to upload before termination.\n"
"         Restart is impossible...starting backup from beginning."
msgstr "NEUSTART: Das Hochladen des ersten Datenträgers ist fehlgeschlagen.\n"
"         Neustart nicht möglich, Sicherung wird vom Anfang wiederaufgenommen."

#: ../duplicity/dup_main.py:1487
#, python-format
msgid "RESTART: Volumes %d to %d failed to upload before termination.\n"
"         Restarting backup at volume %d."
msgstr "NEUSTART: Das Hochladen der Datenträger %d bis %d ist fehlgeschlagen.\n"
"        Sicherung wird mit Datenträger %d wiederaufgenommen."

#: ../duplicity/dup_main.py:1498
#, python-format
msgid "RESTART: Impossible backup state: manifest has %d vols, remote has %d vols.\n"
"         Restart is impossible ... duplicity will clean off the last partial\n"
"         backup then restart the backup from the beginning."
msgstr "NEUSTART: Unmöglicher Sicherungszustand:\n"
"Manifest hat eine Größe von %d, Hintergrundprogramm hat eine Größe von %d.\n"
"        Neustart nicht möglich … duplicity wird die letzte unvollständige\n"
"        Sicherung entfernen und von vorn anfangen."

#: ../duplicity/dup_main.py:1523
msgid "PYTHONOPTIMIZE in the environment causes duplicity to fail to\n"
"recognize its own backups.  Please remove PYTHONOPTIMIZE from\n"
"the environment and rerun the backup.\n\n"
"See https://bugs.launchpad.net/duplicity/+bug/931175\n"
msgstr ""

#: ../duplicity/dup_main.py:1549
#, python-format
msgid "Acquiring lockfile %s"
msgstr "Die Sperrdatei %s wird erfasst"

#: ../duplicity/dup_main.py:1609
#, python-brace-format
msgid "Last {action} backup left a partial set, restarting."
msgstr ""

#: ../duplicity/dup_main.py:1613
#, python-brace-format
msgid "Cleaning up previous partial {action} backup set, restarting."
msgstr ""

#: ../duplicity/dup_main.py:1627
msgid "Last full backup date: none"
msgstr "Letzte vollständige Sicherung: Keine"

#: ../duplicity/dup_main.py:1634
msgid "Last full backup is too old, forcing full backup"
msgstr "Letzte vollständige Sicherung ist zu alt, vollständige Sicherung wird erzwungen"

#: ../duplicity/dup_main.py:1690
msgid "When using symmetric encryption, the signing passphrase must equal the encryption passphrase."
msgstr "Bei symmetrischer Verschlüsselung müssen die Passphrasen für Signatur und Verschlüsselung übereinstimmen."

#: ../duplicity/asyncscheduler.py:259
#, python-format
msgid "task execution done (success: %s)"
msgstr "Aufgabenausführung abgeschlossen (erfolgreich: %s)"

#: ../duplicity/gpginterface.py:694
#, python-brace-format
msgid "GPG process {process.pid} terminated before wait()"
msgstr ""

#: ../duplicity/robust.py:84
#, python-format
msgid "Error listing directory %s"
msgstr "Fehler beim Auflisten des Verzeichnisses %s"

#: ../duplicity/dup_collections.py:200 ../duplicity/dup_collections.py:209
#, python-format
msgid "BackupSet.delete: missing %s"
msgstr "Sicherungssatz löschen: %s fehlt"

#: ../duplicity/dup_collections.py:235
msgid "Fatal Error: No manifests found for most recent backup"
msgstr "Kritischer Fehler: Kein Manifest zur letzten Sicherung gefunden"

#: ../duplicity/dup_collections.py:248
msgid "Fatal Error: Remote manifest does not match local one.  Either the remote backup set or the local archive directory has been corrupted."
msgstr "Kritischer Fehler: Manifest im Hintergrundprogramm stimmt nicht mit der lokalen Version überein. Entweder der entfernte Sicherungssatz oder das lokale Archivverzeichnis ist fehlerhaft."

#: ../duplicity/dup_collections.py:256
#, python-format
msgid "Error processing remote manifest (%s): %s"
msgstr "Fehler beim Verarbeiten des entfernten Manifest (%s): %s"

#: ../duplicity/dup_collections.py:265
msgid "Fatal Error: Neither remote nor local manifest is readable."
msgstr "Kritischer Fehler: Weder das entfernte noch das lokale Manifest ist lesbar."

#: ../duplicity/dup_collections.py:276
#, python-format
msgid "Processing local manifest %s (%s)"
msgstr "Verarbeitung des lokalen Manifest %s (%s)"

#: ../duplicity/dup_collections.py:291
msgid "Error processing remote file ({os.fsdecode(remote_file)}): {util.uexc(message)}"
msgstr ""

#: ../duplicity/dup_collections.py:293
msgid "Processing remote file {os.fsdecode(remote_file)} ({len(remote_file_buffer)})"
msgstr ""

#: ../duplicity/dup_collections.py:311
msgid "No Jsonstat file found, return enmty."
msgstr ""

#: ../duplicity/dup_collections.py:404
msgid "Preferring Backupset over previous one!"
msgstr "Sicherungssatz wird dem Vorherigen vorgezogen!"

#: ../duplicity/dup_collections.py:408
#, python-format
msgid "Ignoring incremental Backupset (start_time: %s; needed: %s)"
msgstr "Schrittweiser Sicherungssatz wird vernachlässigt (Startzeit: %s; benötigt: %s)"

#: ../duplicity/dup_collections.py:417
#, python-format
msgid "Added incremental Backupset (start_time: %s / end_time: %s)"
msgstr "Schrittweiser Sicherungssatz wurde hinzugefügt (Startzeit: %s; Endzeit: %s)"

#: ../duplicity/dup_collections.py:490
msgid "Chain start time: "
msgstr "Erste Sicherung in dieser Kette: "

#: ../duplicity/dup_collections.py:491
msgid "Chain end time: "
msgstr "Letzte Sicherung in dieser Kette: "

#: ../duplicity/dup_collections.py:492
#, python-format
msgid "Number of contained backup sets: %d"
msgstr "Anzahl der enthaltenen Sicherungssätze: %d"

#: ../duplicity/dup_collections.py:493
#, python-format
msgid "Total number of contained volumes: %d"
msgstr "Gesamtanzahl der enthaltenen Datenträger: %d"

#: ../duplicity/dup_collections.py:494 ../duplicity/dup_collections.py:1308
msgid "Type of backup set:"
msgstr "Typ des Sicherungssatzes:"

#: ../duplicity/dup_collections.py:494 ../duplicity/dup_collections.py:1308
msgid "Time:"
msgstr "Zeit:"

#: ../duplicity/dup_collections.py:494
msgid "Num volumes:"
msgstr "Datenträgeranz.:"

#: ../duplicity/dup_collections.py:499 ../duplicity/dup_collections.py:1315
msgid "Full"
msgstr "Vollständig"

#: ../duplicity/dup_collections.py:502 ../duplicity/dup_collections.py:1317
msgid "Incremental"
msgstr "Schrittweise"

#: ../duplicity/dup_collections.py:563
msgid "local"
msgstr "Lokal"

#: ../duplicity/dup_collections.py:726
msgid "Collection Status"
msgstr "Sammlungszustand"

#: ../duplicity/dup_collections.py:728
#, python-format
msgid "Connecting with backend: %s"
msgstr "Mit Hintergrundprogramm wird verbunden: %s"

#: ../duplicity/dup_collections.py:729
#, python-format
msgid "Archive dir: %s"
msgstr "Archivverzeichnis: %s"

#: ../duplicity/dup_collections.py:732
#, python-format
msgid "Found %d secondary backup chain(s)."
msgstr ""

#: ../duplicity/dup_collections.py:734
#, python-format
msgid "Secondary chain %d of %d:"
msgstr "Zusätzliche Kette %d von %d:"

#: ../duplicity/dup_collections.py:739
msgid "Found primary backup chain with matching signature chain:"
msgstr "Primäre Sicherungskette mit passender Signaturkette gefunden:"

#: ../duplicity/dup_collections.py:742
msgid "No backup chains with active signatures found"
msgstr "Keine Sicherungsketten mit aktiven Signaturen gefunden"

#: ../duplicity/dup_collections.py:745
#, python-format
msgid "Also found %d backup set(s) not part of any chain,"
msgstr ""

#: ../duplicity/dup_collections.py:746
#, python-format
msgid "and %d incomplete backup set(s)."
msgstr ""

#. "cleanup" is a hard-coded command, so do not translate it
#: ../duplicity/dup_collections.py:748
msgid "These may be deleted by running duplicity with the \"cleanup\" command."
msgstr "Sie können diese entfernen, indem Sie duplicity mit dem »cleanup«-Befehl starten."

#: ../duplicity/dup_collections.py:750
msgid "No orphaned or incomplete backup sets found."
msgstr "Keine verwaisten oder unvollständigen Sicherungssätze gefunden."

#: ../duplicity/dup_collections.py:766
#, python-format
msgid "%d file(s) exists on backend"
msgstr ""

#: ../duplicity/dup_collections.py:772
#, python-format
msgid "%d file exists in cache"
msgid_plural "%d files exist in cache"
msgstr[0] ""
msgstr[1] ""

#: ../duplicity/dup_collections.py:831
msgid "Warning, discarding last backup set, because of missing signature file."
msgstr "Achtung, der letzte Sicherungssatz wird verworfen, weil die Signaturdatei fehlt."

#: ../duplicity/dup_collections.py:854
msgid "Warning, found the following local orphaned signature file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:862
msgid "Warning, found the following remote orphaned signature file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:870
msgid "Warning, found signatures but no corresponding backup files"
msgstr "Achtung, es wurden Signaturen ohne dazugehörige Sicherungsdaten gefunden"

#: ../duplicity/dup_collections.py:876
msgid "Warning, found incomplete backup sets, probably left from aborted session"
msgstr "Achtung, es wurden unvollständige Sicherungssätze gefunden, wahrscheinlich von einer abgebrochenen Sitzung zurückgelassen"

#: ../duplicity/dup_collections.py:882
msgid "Warning, found the following orphaned backup file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:897
#, python-format
msgid "Extracting backup chains from list of files: %s"
msgstr "Sicherungsketten aus der Liste der Dateien werden entnommen: %s"

#: ../duplicity/dup_collections.py:908
#, python-format
msgid "File %s is part of known set"
msgstr "Datei %s ist Teil eines bekannten Satzes"

#: ../duplicity/dup_collections.py:911
#, python-format
msgid "File %s is not part of a known set; creating new set"
msgstr "Datei %s ist nicht Teil eines bekannten Sicherungssatzes; neuer Satz wird erstellt"

#: ../duplicity/dup_collections.py:916
#, python-format
msgid "Ignoring file (rejected by backup set) '%s'"
msgstr "Datei »%s« wird vernachlässigt (vom Sicherungssatz abgelehnt)"

#: ../duplicity/dup_collections.py:932
#, python-format
msgid "Found backup chain %s"
msgstr "Sicherungskette %s gefunden"

#: ../duplicity/dup_collections.py:937
#, python-format
msgid "Added set %s to pre-existing chain %s"
msgstr "Satz %s zur bestehenden Sicherungskette %s hinzugefügt"

#: ../duplicity/dup_collections.py:940
#, python-format
msgid "Found orphaned set %s"
msgstr "Verwaister Satz %s gefunden"

#: ../duplicity/dup_collections.py:1088
#, python-format
msgid "No signature chain for the requested time. Using oldest available chain, starting at time %s."
msgstr "Keine Signaturkette für die angeforderte Zeit. Nutze die älteste verfügbare Kette, starte bei der Zeit %s."

#: ../duplicity/dup_collections.py:1306
#, python-format
msgid "File: %s"
msgstr "Datei: %s"

#: ../duplicity/dup_collections.py:1307
#, python-format
msgid "Total number of backup: %d"
msgstr "Gesamtzahl an Sicherungen: %d"

#: ../duplicity/dup_collections.py:1308 ../duplicity/dup_collections.py:1344
msgid "Type of file change:"
msgstr "Art der Dateiänderung:"

#: ../duplicity/dup_collections.py:1342
#, python-format
msgid " Backup set time: %s"
msgstr ""

#: ../duplicity/dup_collections.py:1343
#, python-format
msgid "Total number of changes: %d"
msgstr ""

#: ../duplicity/dup_collections.py:1344
msgid "File:"
msgstr ""

#: ../duplicity/patchdir.py:72 ../duplicity/patchdir.py:79
#, python-format
msgid "Patching %s"
msgstr "%s wird repariert"

#: ../duplicity/patchdir.py:547
#, python-format
msgid "Error '%s' patching %s"
msgstr "Fehler »%s« beim Reparieren von %s"

#: ../duplicity/patchdir.py:624
#, python-format
msgid "Writing %s of type %s"
msgstr "Schreibvorgang von %s des Typs %s"

#: ../duplicity/selection.py:115
#, python-format
msgid "Skipping socket %s"
msgstr "Socket %s wird übersprungen"

#: ../duplicity/selection.py:121
#, python-format
msgid "Error initializing file %s"
msgstr "Fehler beim Initialisieren der Datei %s"

#: ../duplicity/selection.py:127 ../duplicity/selection.py:167
#: ../duplicity/selection.py:615
#, python-format
msgid "Error accessing possibly locked file %s"
msgstr "Fehler beim Zugriff auf möglicherweise gesperrte Datei %s"

#: ../duplicity/selection.py:182
#, python-format
msgid "Warning: base %s doesn't exist, continuing"
msgstr "Achtung: Basis %s besteht nicht, es wird fortgefahren"

#: ../duplicity/selection.py:184 ../duplicity/selection.py:205
#: ../duplicity/selection.py:208
#, python-format
msgid "Selecting %s"
msgstr "%s wird ausgewählt"

#: ../duplicity/selection.py:263
#, python-format
msgid "                The last file selection option is the filter option %s, which will have no\n"
"                effect as there are no subsequent file selection options. Exiting because\n"
"                this probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:278
#, python-format
msgid "                Only these filter mode options were specified:\n"
"                    %s\n"
"                Case sensitive globbing is the default behaviour and so this has no effect.\n"
"                Exiting because this probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:346
#, python-format
msgid "                Fatal Error: The file specification\n"
"                    %s\n"
"                cannot match any files in the base directory\n"
"                    %s\n"
"                Useful file specifications begin with the base directory or some\n"
"                pattern (such as '**') which matches the base directory."
msgstr ""

#: ../duplicity/selection.py:360
#, python-format
msgid "Fatal Error while processing expression\n"
"%s"
msgstr "Kritischer Fehler beim Verarbeiten des Ausdrucks\n"
"%s"

#: ../duplicity/selection.py:378
#, python-format
msgid "Reading files to backup from list %s"
msgstr ""

#: ../duplicity/selection.py:403
#, python-format
msgid "                Files-from list contains the absolute path:\n"
"                    %s\n"
"                All paths specified in a files-from list must be given relative to the backup\n"
"                source path."
msgstr ""

#: ../duplicity/selection.py:418
msgid "                Files-from list specified which contains no files, the backup will be empty as\n"
"                a result. Exiting as this probably isn't what you meant,"
msgstr ""

#: ../duplicity/selection.py:435
#, python-format
msgid "                Last selection expression:\n"
"                    %s\n"
"                only specifies that files be included.  Because the default is to\n"
"                include all files, the expression is redundant.  Exiting because this\n"
"                probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:497
#, python-format
msgid "Reading %s filelist %s"
msgstr ""

#: ../duplicity/selection.py:538
#, python-format
msgid "Error compiling regular expression %s"
msgstr "Fehler beim Auswerten des regulären Ausdrucks %s"

#: ../duplicity/lazy.py:349
#, python-format
msgid "Warning: oldindex %s >= newindex %s"
msgstr "Achtung: Alt-Index %s >= Neu-Index %s"

#: ../duplicity/lazy.py:424
#, python-format
msgid "Error '%s' processing %s"
msgstr "Fehler »%s« beim Verarbeiten von %s"

#: ../duplicity/lazy.py:436
#, python-format
msgid "Skipping %s because of previous error"
msgstr "%s wird wegen vorherigem Fehler übersprungen"

#: ../duplicity/path.py:109
#, python-format
msgid "Warning: %s invalid devnums (0x%X), treating as (0, 0)."
msgstr "Achtung: %s ungültige Gerätenummern (0x%X), diese werden wie (0, 0) behandelt."

#: ../duplicity/path.py:239 ../duplicity/path.py:299
#, python-format
msgid "Warning: %s has negative mtime, treating as 0."
msgstr "Achtung: %s hat eine negative mtime, diese wird wie 0 behandelt."

#: ../duplicity/path.py:372
#, python-format
msgid "New file %s"
msgstr "Neue Datei %s"

#: ../duplicity/path.py:375
#, python-format
msgid "File %s is missing"
msgstr "Datei %s fehlt"

#: ../duplicity/path.py:378
#, python-format
msgid "File %%s has type %s, expected %s"
msgstr "Datei %%s ist vom Typ %s, erwartet wurde %s"

#: ../duplicity/path.py:383 ../duplicity/path.py:412
#, python-format
msgid "File %%s has permissions %s, expected %s"
msgstr "Datei %%s hat die Zugriffsrechte %s, erwartet wurde %s"

#: ../duplicity/path.py:389
#, python-format
msgid "File %%s has mtime %s, expected %s"
msgstr "Datei %%s hat mtime %s, erwartet wurde %s"

#: ../duplicity/path.py:400
#, python-format
msgid "Data for file %s is different"
msgstr "Daten für Datei %s sind unterschiedlich"

#: ../duplicity/path.py:408
#, python-format
msgid "Symlink %%s points to %s, expected %s"
msgstr "Symbolischer Verweis %%s zeigt auf %s, erwartet wurde %s"

#: ../duplicity/path.py:415
#, python-format
msgid "Device file %%s has numbers %s, expected %s"
msgstr "Gerätedatei %%s hat Nummern %s, erwartet wurde %s"

#: ../duplicity/path.py:605
#, python-format
msgid "Making directory %s"
msgstr "Verzeichnis %s wird erstellt"

#: ../duplicity/path.py:615
#, python-format
msgid "Deleting %s"
msgstr "»%s« wird gelöscht"

#: ../duplicity/path.py:624
#, python-format
msgid "Touching %s"
msgstr "Zugriffszeit %s wird erstellt oder aktualisiert"

#: ../duplicity/path.py:632
#, python-format
msgid "Deleting tree %s"
msgstr "Baumstruktur %s wird gelöscht"

#: ../duplicity/cli_util.py:48
msgid "Enter 'duplicity --help' for help screen."
msgstr ""

#: ../duplicity/cli_util.py:127
msgid "Running in 'ignore errors' mode due to --ignore-errors.\n"
"Please reconsider if this was not intended"
msgstr ""

#: ../duplicity/cli_util.py:139
msgid "Use of the --asynchronous-upload option is experimental and not safe for production! There are reported cases of undetected data loss during upload. Be aware and periodically verify your backups to be safe."
msgstr ""

#: ../duplicity/cli_util.py:152
msgid "'{val}' is not an int: {str(e)}"
msgstr ""

#: ../duplicity/cli_util.py:166
#, python-brace-format
msgid "'{val} is not a single character."
msgstr ""

#: ../duplicity/cli_util.py:193
#, python-brace-format
msgid "Source should be pathname, not url.  Got '{val}' instead."
msgstr ""

#: ../duplicity/cli_util.py:195
#, python-brace-format
msgid "Argument source_path '{val}' does not exist."
msgstr ""

#: ../duplicity/cli_util.py:201
#, python-brace-format
msgid "Source should be url, not directory.  Got '{val}' instead."
msgstr ""

#: ../duplicity/cli_util.py:207
#, python-brace-format
msgid "Target should be directory, not url.  Got '{val}' instead."
msgstr ""

#: ../duplicity/cli_util.py:212
msgid "Unable to create target dir '{val}': {str(e)}"
msgstr ""

#: ../duplicity/cli_util.py:218
#, python-brace-format
msgid "Target should be url, not directory.  Got '{val}' instead."
msgstr ""

#: ../duplicity/cli_util.py:263
msgid "Verbosity must be one of: digit [0-9], character [ewnid],\n"
"or word ['error', 'warning', 'notice', 'info', 'debug'].\n"
"The default is 3 (Notice).  It is strongly recommended\n"
"that verbosity level is set at 2 (Warning) or higher."
msgstr ""

#: ../duplicity/cli_util.py:399
#, python-brace-format
msgid "Specified archive directory '{archive_dir_path.uc_name}' is not a directory"
msgstr ""

#: ../duplicity/cli_util.py:408
msgid "Encrypt key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr ""

#: ../duplicity/cli_util.py:428
msgid "Hidden dncrypt key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr ""

#: ../duplicity/cli_util.py:442
msgid "Sign key should be an 8, 16, or 40 character hex string, like 'AA0E73D2'.\n"
msgstr ""

#: ../duplicity/argparse311.py:291
msgid "usage: "
msgstr ""

#: ../duplicity/argparse311.py:761
#, python-format
msgid "argument %(argument_name)s: %(message)s"
msgstr ""

#: ../duplicity/argparse311.py:870
msgid ".__call__() not defined"
msgstr ""

#: ../duplicity/argparse311.py:1115
#, python-format
msgid "conflicting subparser: %s"
msgstr ""

#: ../duplicity/argparse311.py:1118
#, python-format
msgid "conflicting subparser alias: %s"
msgstr ""

#: ../duplicity/argparse311.py:1152
#, python-format
msgid "unknown parser %(parser_name)r (choices: %(choices)s)"
msgstr ""

#: ../duplicity/argparse311.py:1215
#, python-format
msgid "argument \"-\" with mode %r"
msgstr ""

#: ../duplicity/argparse311.py:1223
#, python-format
msgid "can't open '%(filename)s': %(error)s"
msgstr ""

#: ../duplicity/argparse311.py:1427
#, python-format
msgid "cannot merge actions - two groups are named %r"
msgstr ""

#: ../duplicity/argparse311.py:1462
msgid "'required' is an invalid argument for positionals"
msgstr ""

#: ../duplicity/argparse311.py:1483
#, python-format
msgid "invalid option string %(option)r: must start with a character %(prefix_chars)r"
msgstr ""

#: ../duplicity/argparse311.py:1500
#, python-format
msgid "dest= is required for options like %r"
msgstr ""

#: ../duplicity/argparse311.py:1517
#, python-format
msgid "invalid conflict_resolution value: %r"
msgstr ""

#: ../duplicity/argparse311.py:1534
#, python-format
msgid "conflicting option string: %s"
msgid_plural "conflicting option strings: %s"
msgstr[0] ""
msgstr[1] ""

#: ../duplicity/argparse311.py:1595
msgid "mutually exclusive arguments must be optional"
msgstr ""

#: ../duplicity/argparse311.py:1670
msgid "positional arguments"
msgstr ""

#: ../duplicity/argparse311.py:1689
msgid "show this help message and exit"
msgstr ""

#: ../duplicity/argparse311.py:1721
msgid "cannot have multiple subparser arguments"
msgstr ""

#: ../duplicity/argparse311.py:1769 ../duplicity/argparse311.py:2264
#, python-format
msgid "unrecognized arguments: %s"
msgstr ""

#: ../duplicity/argparse311.py:1868
#, python-format
msgid "not allowed with argument %s"
msgstr ""

#: ../duplicity/argparse311.py:1912 ../duplicity/argparse311.py:1926
#, python-format
msgid "ignored explicit argument %r"
msgstr ""

#: ../duplicity/argparse311.py:2030
#, python-format
msgid "the following arguments are required: %s"
msgstr ""

#: ../duplicity/argparse311.py:2042
#, python-format
msgid "one of the arguments %s is required"
msgstr ""

#: ../duplicity/argparse311.py:2083
msgid "expected one argument"
msgstr ""

#: ../duplicity/argparse311.py:2084
msgid "expected at most one argument"
msgstr ""

#: ../duplicity/argparse311.py:2085
msgid "expected at least one argument"
msgstr ""

#: ../duplicity/argparse311.py:2089
#, python-format
msgid "expected %s argument"
msgid_plural "expected %s arguments"
msgstr[0] ""
msgstr[1] ""

#: ../duplicity/argparse311.py:2143
#, python-format
msgid "ambiguous option: %(option)s could match %(matches)s"
msgstr ""

#: ../duplicity/argparse311.py:2207
#, python-format
msgid "unexpected option string: %s"
msgstr ""

#: ../duplicity/argparse311.py:2402
#, python-format
msgid "%r is not callable"
msgstr ""

#: ../duplicity/argparse311.py:2419
#, python-format
msgid "invalid %(type)s value: %(value)r"
msgstr ""

#: ../duplicity/argparse311.py:2429
#, python-format
msgid "invalid choice: %(value)r (choose from %(choices)s)"
msgstr ""

#: ../duplicity/argparse311.py:2505
#, python-format
msgid "%(prog)s: error: %(message)s\n"
msgstr ""

